src/Zir.zig

```diff
@@ -2919,6 +2919,11 @@ pub const Inst = struct {
         index: u32,
     };
 
+    pub const Panic = struct {
+        force_comptime: u32,
```

Comment:
oof an entire 32 bit integer for 1 bit of information, perhaps could we have `panic_comptime` and `panic` tags instead? I suppose if we're out of ZIR tags then this could be a later enhancement.

Reply:
> perhaps could we have `panic_comptime` and `panic` tags instead?

I'll do that, I don't know why I didn't think of that when making the change.